lib/ipam/ipam.go

```diff
@@ -923,20 +851,20 @@ func (c ipams) SetIPAMConfig(cfg IPAMConfig) error {
 	}
 
 	if !cfg.StrictAffinity && !cfg.AutoAllocateBlocks {
-		return goerrors.New("Cannot disable 'StrictAffinity' and 'AutoAllocateBlocks' at the same time")
+		return errors.New("Cannot disable 'StrictAffinity' and 'AutoAllocateBlocks' at the same time")
```

Comment:
side question, are we planning to expose these config options?

Reply:
nothing in plan to at the moment.  it may be that we decide later on to revisit the IPAM architecture prior to GA - but nothing at the moment.